reddit/uploads/api/views.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from users.models import User
+from serializers import PostSerializer, UserPostsSerializer
+
+
+class UserPostsView(generics.ListAPIView):
+    queryset = User.objects.all()
+    serializer_class = UserPostsSerializer
+
+    def list(self, request, *args, **kwargs):
```

Comment:
The list() method here doesn't really do any custom implementation, why override it at all?